docs/guides/serverless-manage-resources.ipynb

```diff
@@ -108,81 +126,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "%%writefile --append ./source_files/transpile_remote.py\n",
+    "\n",
     "@distribute_task(target={\n",
     "    \"cpu\": 16,\n",
-    "    \"mem\": 32 * 1024 * 1024 * 1024\n",
+    "    \"mem\": 2 * 1024 * 1024 * 1024\n",
```

Comment:
Why this change?

Reply:
The prose says:

>  To allocate 2 GB of memory for your worker, you need to allocate `"mem": 2 * 1024 * 1024 * 1024`

As a reader, I was mildly surprised when the code example allocated 32GB rather than 2.